src/components/PersonDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  person: Person;
+  selectedSlug: string;
+};
+
+export const PersonDetails:FC<Props> = ({ people, person, selectedSlug }) => {
+  const {
+    sex, born, died, motherName, fatherName, slug,
+  } = person;
+
+  const motherFound = people.find(item => item.name === motherName);
+  const motherCell = motherName || '-';
+
+  const fatherFound = people.find(item => item.name === fatherName);
+  const fatherCell = fatherName || '-';
```

Comment:
Consider using 'fatherFound?.name' instead of 'fatherName' for 'fatherCell'. This ensures that the value is '-' when 'fatherFound' is undefined.